src/client/login.html

```diff
@@ -73,11 +81,12 @@
                 <h2 class="form-signin-heading">WebGME sign in</h2>
                 <div class="row">
                     <div class="col-md-6">
-                        <input name="username" type="text" autocomplete="off" autofocus="autofocus" class="input-block-level" placeholder="Username">

```

Comment:
Why do we need to remove autofocus? If we visit the page i can just start typing...